### PR TITLE
[VMRay] fix encoding of file names

### PR DESCRIPTION
**Pull request by jthom-vmray**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Remove forbidden characters from the file name before submitting the file to VMRay.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by RotemAmit — COMMENTED** [review]
Thank you for your contribution! Nice work!
* Can you make a video showing the change that you did and send it to me? You can add it to the PR, send me an email or via DFIR.
* Please update the docker image tag in the YML file to the latest numeric tag and add it to the release notes. You can check for the most updated version of demisto/python3 in this [link](https://hub.docker.com/r/demisto/python3/tags). In order to add the docker image change to the release notes you can use the command demisto-sdk update-release-notes, more explanation can be found [here](https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes).

**Comment by jthom-vmray** [comment]
> Thank you for your contribution! Nice work!
> 
>     * Can you make a video showing the change that you did and send it to me? You can add it to the PR, send me an email or via DFIR.
By the fact that it is a small change, I think a video too much effort to create.
To explain the change in other words:
VMRay Platform can only handle valid Windows filenames. Previously only a backslash was removed from the filename. This made it possible to send files to VMRay that are invalid, e.g., if a file in XSOAR is called `foo_?_bar.exe`.
The change removes all characters that are not allowed in Windows, so that the example filename would look like this: `foo__bar.exe`

File: Packs/VMRay/Integrations/VMRay/VMRay_test.py [hunk 1]
```diff
@@ -53,20 +53,19 @@ def test_upload_sample_command(mocker):
...
+@pytest.mark.parametrize(
+    "file_name, expected",
+    [
+        ("abc.exe", b"abc.exe"),
+        ("<>:\"/\\|?*a.exe", b"a.exe"),
+        ("\\test\\encode\\file\\name", b"testencodefilename"),
+        ("ñá@.exe", b"\xc3\xb1\xc3\xa1@.exe"),
+    ]
+)
+def test_encoding_file_name(file_name, expected):
```
Comment (RotemAmit) [inline Packs/VMRay/Integrations/VMRay/VMRay_test.py:65]: Please add a doc string with Given When, Then, with an updated explanation about the test.

**Review by RotemAmit — COMMENTED** [review]
Thank you for the explanation about the change that you made.
I have just 1 comment and then we are good to go.
Thank you very much for your contribution!

**Review by RotemAmit — APPROVED** [review]

**Pull request merged**